model-optimizer/extensions/front/EmbeddingBagFuse.py

```diff
@@ -0,0 +1,91 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.embedding_bag import EmbeddingBagOffsetsSum
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph, rename_nodes
+
+
+class EmbeddingBagFuse(FrontReplacementSubgraph):
+    enabled = True
+
+    def run_after(self):
+        from extensions.front.ExpandDimsToUnsqueeze import ExpandDimsToUnsqueeze
+        from extensions.front.AttributedGatherNormalizer import AttributedGatherNormalizer
+        return [ExpandDimsToUnsqueeze, AttributedGatherNormalizer]
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('weights', dict(op='Const')),
+                ('concat_before', dict(op='Concat')),
+                ('gather_before1_1', dict(op='Gather')),
```

Comment:
Looks like we don't need '_1' in the end of the names. Also there is just one slice, gather_after, reduce and unsqueeze_after so we can remove '1' in the end of the name.